fapi-client/src/main/scala/com/gu/facia/api/FAPI.scala

```diff
@@ -0,0 +1,80 @@
+package com.gu.facia.api
+
+import com.gu.contentapi.client.GuardianContentClient
+import com.gu.facia.api.contentapi.ContentApi
+import com.gu.facia.api.contentapi.ContentApi.{AdjustItemQuery, AdjustSearchQuery}
+import com.gu.facia.api.models._
+import com.gu.facia.client.ApiClient
+import com.gu.facia.client.models.{CollectionConfigJson, TrailMetaData}
+
+import scala.concurrent.ExecutionContext
+
+
+object FAPI {
```

Comment:
a) because they aren't required everywhere
b) because this way we just have pure functions. Whenever you have code that has a dependency on some other value, there are lots of ways of getting that value in scope. People always end up making traits and classes and so on to get that to happen. But you know how else you can make code in a function depend on a value? With an argument. It's amazingly simple and I'm always baffled people don't just do this everywhere.